app/services/teacher_training_api/import_course.rb

```diff
@@ -6,10 +6,8 @@ class ImportCourse
 
     # For full list, see https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-courseattributes
```

Comment:
Can we update this link? Doesn't go anywhere now.

Reply:
done!